core/domain/state_domain_test.py

```diff
@@ -650,6 +650,21 @@ def test_voiceover_validation(self):
             ):
             with self.swap(audio_voiceover, 'needs_update', 'hello'):
                 audio_voiceover.validate()
+        with self.assertRaisesRegexp(
+            utils.ValidationError, 'Expected duration_secs to be a float'
+            ):
+            with self.swap(audio_voiceover, 'duration_secs', 0):
+                audio_voiceover.validate()
+        with self.assertRaisesRegexp(
+            utils.ValidationError, 'Invalid duration in seconds'
```

Comment:
Use clearer error messages, e.g.: Expected duration_secs to be greater than 0.

(Try to be informative, don't just say "invalid".)

Reply:
Changed the message should say 'Expected duration_secs to be a positive number.'

Reply:
Done via https://github.com/oppia/oppia/pull/8203#discussion_r382933504